browser/src/control/Control.NotebookbarBuilder.js

```diff
@@ -884,12 +884,16 @@ L.Control.NotebookbarBuilder = L.Control.JSDialogBuilder.extend({
 	},
 
 	_insertGraphicControl: function(parentContainer, data, builder) {
-		var options = {hasDropdownArrow: builder.map['wopi'].EnableInsertRemoteImage};
+		var options = {
+			hasDropdownArrow: builder.map['wopi'].EnableInsertRemoteImage && !builder.map['wopi'].DisableInsertLocalImage
+		};
 		var control = builder._unoToolButton(parentContainer, data, builder, options);
 
 		$(control.container).unbind('click.toolbutton');
 		$(control.container).click(function () {
-			if (builder.map['wopi'].EnableInsertRemoteImage) {
+			var hasRemote = builder.map['wopi'].EnableInsertRemoteImage;
+			var hasLocal = !builder.map['wopi'].DisableInsertLocalImage;
```

Comment:
We should use either positive or negative statements. Why not make both EnableInsert...Image ?

Reply:
I can rename it `EnableInsertLocalImage` and have it set to `true` by default. I thought `false` by default was fine, but either work.

Reply:
@Ashod  @caolanm There was a very good reason why the existing is `Enable` and this is `Disable`. `EnableInsertRemoteImage` concerns inserting images from the WOPI host, and needs additional support on the side of the WOPI host, while this one is for disabling a feature in COOL, because an integration doesn't want to allow inserting images from the users' computer.
If this patch has to go in, I would strongly suggest considering reverting to the original naming.

Reply:
It's back to `DisableInsertLocalImage`